src/panels/experimental-ui/hui-glance-card.js

```diff
@@ -0,0 +1,136 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import computeStateDisplay from '../../common/entity/compute_state_display.js';
+import computeStateName from '../../common/entity/compute_state_name.js';
+
+import '../../components/entity/state-badge.js';
+import '../../components/ha-card.js';
+
+import EventsMixin from '../../mixins/events-mixin.js';
+import LocalizeMixin from '../../mixins/localize-mixin.js';
+
+/*
+ * @appliesMixin EventsMixin
+ * @appliesMixin LocalizeMixin
+ */
+class HuiGlanceCard extends LocalizeMixin(EventsMixin(PolymerElement)) {
+  static get template() {
+    return html`
+      <style>
+        ha-card {
+          padding: 16px;
+        }
+        .header {
+          @apply --paper-font-headline;
+          /* overwriting line-height +8 because entity-toggle can be 40px height,
+            compensating this with reduced padding */
+          line-height: 40px;
+          color: var(--primary-text-color);
+          padding: 4px 0 12px;
+        }
+        .header .name {
+          @apply --paper-font-common-nowrap;
+        }
+        .entities {
+          padding: 4px 0;
+          display: flex;
+          flex-wrap: wrap;
+          flex-basis: 20%;
```

Comment:
This still seems very aimed at 5 entities 😉 I'm fine with that for the initial release, but it's my understanding that if we drop this and the `width` from `.entity`, it should look good.